### PR TITLE
Update SKYSTARSF405AIO 

### PR DESCRIPTION
**Pull request by howels**
Update SKYSTARSF405AIO as production board will use updated gyro orientation

### CODE REVIEW

File: configs/SKYSTARSF405AIO/config.h [hunk 1]
```diff
@@ -105,6 +106,5 @@
 #define FLASH_SPI_INSTANCE SPI3
 #define USE_SPI_GYRO
 #define GYRO_1_SPI_INSTANCE SPI1
-#define GYRO_1_ALIGN CW180_DEG
-#define GYRO_1_ALIGN_YAW 1800
-
+#define GYRO_1_ALIGN CW0
```
Comment (haslinghuis) [inline configs/SKYSTARSF405AIO/config.h:109]:
```suggestion
```

[the remark this answers is not in the document]
Comment (howels) [reply configs/SKYSTARSF405AIO/config.h:109]: Thanks, wasn't sure if the default was good as I saw some hangs when setting it in configurator

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
need update unified as well?
https://github.com/betaflight/unified-targets/blob/master/configs/default/SKST-SKYSTARSF405AIO.config#L118

**Comment by nerdCopter** [comment]
can we revert BMI270 for those of use that have prototype board. less jumping hoops.

**Pull request merged**